pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -101,16 +99,11 @@ findDependency() {
 }
 
 autoPatchelfFile() {
-    local dep rpath="" toPatch="$1"
+    local dep interpreter missing rpath="" toPatch="$1"
```

Comment:
Is this also a thing that `shellcheck` has reported or did you just prefer to declare all `local`s at the top of the function?